packages/cubejs-base-driver/src/BaseDriver.ts

```diff
@@ -299,6 +333,35 @@ export abstract class BaseDriver implements DriverInterface {
     }
   }
 
+  public getSchemas() {
```

Comment:
look at the driver capabilities method.
add some kind of `icrementalSchemaLoading` flag to handle

Reply:
expose driver capabilities